src/BrazeMessageComponent.tsx

```diff
@@ -25,8 +26,10 @@ type CommonComponentProps = {
     brazeMessageProps: BrazeMessageProps;
 };
 
+export type BrazeComponent<P> = React.FC<P>;
```

Comment:
This type doesn't feel like it's pulling its weight any longer! Since we're not extending it any more. What do you think about losing it?